libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/editor/DBTablePartitionEditor.java

```diff
@@ -76,13 +76,9 @@ public String generateCreateDefinitionDDL(@NotNull DBTablePartition partition) {
         if (Objects.nonNull(partition.getSubpartitionTemplated()) && partition.getSubpartitionTemplated()) {
             generateCreateTemplateSubPartition(partition, sqlBuilder);
         }
-        if (Objects.nonNull(partition.getPartitionOption().getPartitionsNum())) {
-            sqlBuilder.append("\nPARTITIONS ")
-                    .append(String.valueOf(partition.getPartitionOption().getPartitionsNum()));
-            if (partition.getPartitionOption().getType() == DBTablePartitionType.HASH
-                    || partition.getPartitionOption().getType() == DBTablePartitionType.KEY) {
-                return sqlBuilder.toString();
-            }
+        if (partition.getPartitionOption().getType() == DBTablePartitionType.HASH
+                || partition.getPartitionOption().getType() == DBTablePartitionType.KEY) {
+            return sqlBuilder.toString();
```

Comment:
If we return directly here, then there will be no partition definition information for hash and key types?